### ISSUE TITLE
has no exported member 'LocalStorageModule' @5.30

### ISSUE THREAD

**Issue by doughlass** [cyrilletuzi/angular-async-local-storage#35]
I've ran `yarn add @ngx-pwa/local-storage@5` and it's installed `5.3.0`

I've added in `app.module.ts`

```import { LocalStorageModule } from '@ngx-pwa/local-storage';```

and adding `LocalStorageModule` to my `@NgModule`

But get the dreaded 

> has no exported member 'LocalStorageModule' 

I've tried without the import line but I get an error in my front end

> core.js:1449 ERROR Error: Uncaught (in promise): Error: StaticInjectorError(AppModule)[CreateComponent -> LocalStorage]:

**Comment by cyrilletuzi** [comment]
@doughlass I can't reproduce this error (I use npm instead of yarn, but it shouldn't matter for the lib).

Is your code using the CLI or a special configuration? Is your code available somewhere so I have a look?

**Comment by cyrilletuzi** [comment]
@doughlass Could you please answer? If it's a real issue, it's a major one and should be fixed as soon as possible.

**Comment by doughlass** [comment]
Just wanted to confirm after several attempts I have managed to get it working by
```

rm -rf node_modules
yarn
```

**Comment by dianjuar** [comment]
I'm having the same problem.

When I run `npm start` I receibe this error 

```
ERROR in : Unexpected value 'LocalStorageModule in /home/dianjuar/ProgramerProjects/User-CRUD-testing-creation/node_modules/@ngx-pwa/local-storage/local-storage.d.ts' imported by the module 'UsersModule in /home/dianjuar/ProgramerProjects/User-CRUD-testing-creation/src/app/users/users.module.ts'. Please add a @NgModule annotation.
```

I have tried, delete the `node_modules` folder with the file `package-lock.json` and `yarn.lock` and trying install with both `npm` and `yarn`

**Comment by cyrilletuzi** [comment]
@dianjuar That is not the same error.

First, do you have the right version of the lib installed (v5 for Angular 5, v6 for Angular 6)?

Second, `LocalStorageModule` only exists for v5. You have to delete it in v6.

Third, it is a module that should be imported only once in `AppModule`. Your error shows you're not importing it in the right place.

If after all these checks it's still not OK, please open another issue with a full reproduction of the problem.
